eiffel-syntax-and-usage/activity-linking.md

```diff
@@ -71,6 +71,23 @@ __Legal targets:__ [EiffelActivityTriggeredEvent][ActT],
 [EiffelTestCaseTriggeredEvent][TCT], [EiffelTestSuiteStartedEvent][TSS]  
 __Multiple allowed:__ Yes  
 
+### ORIGINAL_TRIGGER
+
+This link type is used for sequences of (usually) failing activities where the first activity declares the "root" trigger and the subsequent activities are retries of the first activity.
+
+*The image below shows how a source code change triggers a build activity. That activity fails and is retried, and the second activity (which might be manually triggered by a user) has an ORIGINAL_TRIGGER link to the first activity.*
+
+![An example of using ORIGINAL_TRIGGER](./original-trigger.png "An example of using ORIGINAL_TRIGGER")
```

Comment:
Maybe the picture should show a third activity execution as well, with ORIGINAL_TRIGGER targeting the first ActT, to avoid confusion of whether a third ActT should target the first or the previous ActT? We should probably expect the readers to understand that by the phrasing in the text above though.

Reply:
Yes. Leave the picture as-is then, but maybe rephrase the text slightly. It does state "first activity", but maybe an additional sentence mentioning a third activity also linking to the first activity could be written.

Reply:
Done.